Tests/SonarQube.TeamBuild.PreProcessor.Tests/Infrastructure/PreprocessTestUtils.cs

```diff
@@ -13,18 +13,21 @@ namespace SonarQube.TeamBuild.PreProcessor.Tests
     internal static class PreprocessTestUtils
     {
         /// <summary>
-        /// Creates and returns an environment scope that contains all of the required
-        /// legacy TeamBuild environment variables
+        /// Creates and returns an environment scope configured as if it
+        /// is not running under TeamBuild
         /// </summary>
-        public static EnvironmentVariableScope CreateValidLegacyTeamBuildScope(string tfsUri, string buildUri)
```

Comment:
This method wasn't being used. I've amended it to create a non-TeamBuild environment. The CI build will determine whether the new method actually works as expected...